PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +612,61 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (SourceFile == null)
+                return false;
+
+            string ext = System.IO.Path.GetExtension(SourceFile).ToLowerInvariant();
+
+            // Haxe, ActionScript, and JavaScript support Regex literals
+            if (!Helpers.FileInspector.IsActionScript(SourceFile, ext) &&
```

Comment:
Worth moving this detection to the beginning of the parsing instead of resolving that every time the function is called.